# request params
sensor_msgs/Image color_image
sensor_msgs/Image depth_image
---
# response params
geometry_msgs/Pose pose_msg
